charts/camunda-platform-8.6/templates/camunda/_helpers.tpl

```diff
@@ -441,6 +441,8 @@ Web Modeler templates.
       {{- else -}}
         {{- printf "%s://%s%s" $proto .context.Values.global.ingress.host (index .context.Values.webModeler "contextPath") -}} 
       {{- end -}}
+    {{- else -}}
+      {{- printf "http://localhost:8084" -}}
```

Comment:
This function `getExternalURLModeler` accepts a `component` argument, which could either be `websockets` or `webapp`.  The port you used only accounts for the `webapp`.

I'd suggest something like

```
      {{- if eq .component "websockets" }}
              {{- printf "http://localhost:8085" -}}
      {{- else -}}
              {{- printf "http://localhost:8084" -}}
      {{- end -}}
```